src/test/java/tests/FilterEvalTests.java

```diff
@@ -833,9 +834,25 @@ public void processQualifier_useInvalidUsername_getUsernameWarning() {
                                                             new ArrayList<>(),
                                                             new ArrayList<>(),
                                                             new ArrayList<>(Arrays.asList(user))));
-        verifySemanticException(model, "involves:bob", String.format(USER_WARNING_ERROR_FORMAT, "bob", REPO));
-        verifySemanticException(model, "involves:foxx", String.format(USER_WARNING_ERROR_FORMAT, "bob", REPO));
-        verifySemanticException(model, "author:alice", String.format(USER_WARNING_ERROR_FORMAT, "alice", REPO));
+        verifyUserWarning(model, "involves:bOb",
+                            Arrays.asList(String.format(USER_WARNING_ERROR_FORMAT, "bOb", REPO)));
+        verifyUserWarning(model, "involves:foxX",
+                            Arrays.asList(String.format(USER_WARNING_ERROR_FORMAT, "foxX", REPO)));
+        verifyUserWarning(model, "author:alice",
+                            Arrays.asList(String.format(USER_WARNING_ERROR_FORMAT, "alice", REPO)));
+    }
+
+    @Test
+    public void processQualifier_useValidUsername_noUsernameWarning() {
+        TurboUser user = new TurboUser(REPO, "fox", "charlie");
+        IModel model = TestUtils.singletonModel(new Model(REPO,
+                                                            new ArrayList<>(),
+                                                            new ArrayList<>(),
+                                                            new ArrayList<>(),
+                                                            new ArrayList<>(Arrays.asList(user))));
```

Comment:
As the same thing is used in another test below, you can even extract it to a utility method.